source/help/messaging/organizing-conversations.rst

```diff
@@ -0,0 +1,69 @@
+Organizing Conversations using Collapsed Reply Threads (Beta)
+=============================================================
+
+Threads are a key part of the messaging experience in Mattermost. Threads organize conversations within channels, and threads enable users to discuss topics without adding noise to channels or Direct Messages. 
+
+Collapsed Reply Threads (Beta) offers an enhanced experience for users communicating in threads and replying to messages. Our goal is to improve users’ ability to process channel content, find, follow, and resume conversations more easily, and keep threaded conversations focused.
+
+.. image:: ../../images/collapsed-reply-threads.gif
+  :alt: Organize conversations using Collapsed Reply Threads.
+
+Your System Admin must enable the Collapsed Reply Threads feature in the System Console. See our `Configuration Settings <https://docs.mattermost.com/administration/config-settings.html#collapsed-reply-threads-beta>`__ documentation for details. You can `enable this feature for your Mattermost account <https://docs.mattermost.com/help/settings/account-settings.html#collapsed-reply-threads-beta>`__ by going to **Account Settings > Display > Collapsed Reply Threads (Beta)**.
+
+Known Issues
+------------
+
+.. important::
+
+    Collapsed Reply Threads are available as an early beta in Mattermost Cloud and Mattermost Server v5.37 and later. It's expected that you may experience bugs as we stabilize the feature. In particular, please be aware of these known issues: 
+    
+    - When enabling Collapsed Reply Threads for the first time, you may see channels or threads you’ve seen before appear as unread. To resolve this,
+      - View any unread channels or use ALT+SHIFT+UP/DOWN to switch to unread channels with the keyboard. Work in-progress ticket: `MM-35494 <https://mattermost.atlassian.net/browse/MM-35494>`__
+      - Use the **Mark all as read** button in the **Threads** view to mark your threads as read. Work in-progress ticket: `MM-35345 <https://mattermost.atlassian.net/browse/MM-35345>`__
+    - You may experience lag in your desktop or web client if you're following many threads or opening threads with many replies. Work in-progress tickets: `MM-36696 <https://mattermost.atlassian.net/browse/MM-36696>`__, `MM-36697 <https://mattermost.atlassian.net/browse/MM-36697>`__, `MM-36698 <https://mattermost.atlassian.net/browse/MM-36698>`__
+    - You should expect to see more unread channels and notifications on your mobile device than you'll see on your Desktop instance of Mattermost. This is expected until mobile support is available for Collapsed Reply Threads. Mobile support for Collapsed Reply Threads is expected next month. For early access, join our `mobile beta program <https://github.com/mattermost/mattermost-mobile#testing>`__. 
```

Comment:
...until mobile support is available for Collapsed Reply Threads, expected in August 2021.

Proposal to eliminate repetition? ^